draft-ietf-taps-interface.md

```diff
@@ -2281,9 +2281,12 @@ Default:
 
 Numeric values of these properties specify an upper-bound rate that a transfer is not expected to
 exceed (even if flow control and congestion control allow higher rates), and/or a
-lower-bound rate below which the application does not deem
-it will be useful. These are specified in bits per second.
-The enumerated value `Unlimited` indicates that no bound is specified.
+lower-bound application-layer rate below which the application does not deem
+it will be useful. These rate values are meant at the application layer, i.e. not considering header overheads
+from protocols used by the Transport Services system. The values are specified in bits per second,
```

Comment:
```suggestion
it will be useful. These rate values are measured at the application layer, i.e. not considering header overheads
from protocols used by the Transport Services system. The values are specified in bits per second,
```

Reply:
Added a small delta to the suggestion from Tommy